packages/clay-card/src/CardWithFolder.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayCard from './Card';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+
+interface IProps {
+	actions?: React.ComponentProps<typeof ClayDropDownWithBasicItems>['items'];
+	/**
+	 * Path or URL to folder
+	 */
+	href?: string;
+
+	/**
+	 * Name of the folder
+	 */
+	name: string;
+
+	/**
+	 * Callback for when item is selected
+	 */
+	onSelectChange?: (val: boolean) => void;
+
+	/**
+	 * Flag to indicate if card is selected
+	 */
+	selected?: boolean;
+
+	/**
+	 * Path to clay icon spritemap
+	 */
+	spritemap: string;
+}
+
+export const ClayCardWithFolder: React.FunctionComponent<IProps> = ({
+	actions,
+	href,
+	name,
+	onSelectChange,
+	selected = false,
+	spritemap,
+}) => {
+	const content = (
+		<ClayCard.Body>
+			<div className="autofit-col">
+				<ClaySticker inline>
+					<ClayIcon spritemap={spritemap} symbol="folder" />
```

Comment:
I would say the same here

Reply:
I'd be hesitant to make it configurable since its explicitly the "CardWithFolder" component. If we remove that symbol, it starts to be much more vague.

Reply:
Yeah, you're right about that, you might want to rename it `CardWithHorizontal` to follow what Lexicon said: https://docs.google.com/document/d/1st2moaRVphrsFrDJUz-ic26aH-V46YkQam75X6BHPY0/edit#heading=h.x89wughhxazs

What do you think?